src/quo2/components/selectors/selectors.cljs

```diff
@@ -0,0 +1,129 @@
+(ns quo2.components.selectors.selectors
+  (:require [reagent.core :as reagent]
+            [quo.react :as react]
+            [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [cljs-bean.core :as bean]
+            [quo.theme :as theme]
+            [quo2.components.icon :as icons]))
+
+(defn get-color [value disabled?]
+  (if value
+    (colors/theme-colors
+     (colors/custom-color :primary 50 (when disabled? 30))
+     (colors/custom-color :primary 60 (when disabled? 30)))
+    (colors/theme-colors
+     (colors/alpha colors/neutral-20 (if disabled? 0.4 1))
+     (colors/alpha colors/neutral-70 (if disabled? 0.3 1)))))
+
+(defn checkbox-prefill-view [props]
+  (let [{:keys [value onChange disabled? containerStyle]} (bean/bean props)]
+    (reagent/as-element
+     [rn/touchable-without-feedback
+      {:on-press (when (and onChange (not disabled?)) onChange)}
+      [rn/view
+       {:style        (merge
+                       (js->clj containerStyle)
+                       {:height 21
+                        :width 21
+                        :border-radius 6
+                        :background-color (colors/theme-colors
+                                           (colors/alpha colors/neutral-20 (if disabled? 0.3 1))
+                                           (colors/alpha colors/neutral-70 (if disabled? 0.3 1)))})
+        :accessibility-label (str "checkbox-" (if value "on" "off"))
+        :accessibility-role  :checkbox}
+       (when value
+         [rn/view {:style {:height 20
+                           :width 20}}
+          [icons/icon :main-icons2/check
+           {:size 20
+            :color (colors/theme-colors
+                    (colors/alpha colors/neutral-100 (if disabled? 0.3 1))
+                    (colors/alpha colors/white (if disabled? 0.3 1)))}]])]])))
+
+(defn checkbox-view [props]
+  (let [{:keys [value onChange disabled? containerStyle]} (bean/bean props)]
+    (reagent/as-element
+     [rn/touchable-without-feedback
+      {:on-press (when (and onChange (not disabled?)) onChange)}
+      [rn/view
+       {:style        (merge
+                       (js->clj containerStyle)
+                       {:height (if value 20 21)
+                        :width (if value 20 21)
+                        :border-radius 6
+                        :border-width (if value 0 1)
+                        :background-color (if value
+                                            (get-color value disabled?)
+                                            (colors/theme-colors
+                                             colors/white
+                                             (colors/alpha colors/neutral-80 0.4)))
```

Comment:
we already have [color](https://github.com/status-im/status-mobile/blob/b4d95b240d88066c34c0e8fcd966a214ac0b0912/src/quo2/foundations/colors.cljs#L62) for this configuration.

Reply:
yep, will adjust